components/table/Column.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+class Column extends React.Component {
+  render() {
+    const { context, getter, id } = this.props
+    const value = getter ? getter(context) : context[id]
+    return value !== undefined ? value : null
```

Comment:
because it makes the rendering of table columns [much easier](https://github.com/oacore/dashboard/blob/table/pages/data-providers/%5Bdata-provider-id%5D/content.jsx#L100)...

Reply:
Yes, sorry for this comment. I found this later. I read files one by one during review what is not always good sequence.